README.md

```diff
@@ -1,65 +1,73 @@
-README
-======
+<p align="center"><a href="https://symfony.com" target="_blank">
+    <img src="http://symfony.com/logos/symfony_black_02.svg">
```

Comment:
@stof maybe weird, but stable ;-)

@javiereguiluz Ok, I understand, but maybe it should be dedicated resource (`symfony_repo_header.svg` instead of `symfony_black_02.svg`)? As far as everyone knows the `symfony_black_02.svg` is displayed here it's ok, but file could be renamed or deleted.